libres/lib/enkf/enkf_fs.cpp

```diff
@@ -361,8 +361,7 @@ static enkf_fs_type *enkf_fs_mount_block_fs(FILE *fstab_stream,
 }
 
 enkf_fs_type *enkf_fs_create_fs(const char *mount_point,
-                                fs_driver_impl driver_id, void *arg,
-                                bool mount) {
+                                fs_driver_impl driver_id, bool mount) {
     /*
```

Comment:
Maybe change this to `//` line comments and update indentation while we are at it.